src/astarte-charts/react/PieChart/index.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import React, { useMemo } from 'react';
+import { Pie } from 'react-chartjs-2';
+import Color from 'color';
+import type {
+  AstarteChartProvider,
+  AstarteChartSingleValue,
+  AstarteChartValueObject,
+  AstarteChartValueObjectTimestamped,
+} from 'astarte-charts';
+
+import { useChartProviders } from '../hooks';
+
+const chartOptions = {
+  responsive: true,
+  legend: {
+    position: 'right',
```

Comment:
In the future we may as well support any kind of specific options.
For the sake of simplicity, here I just added a `showLegend: boolean` prop so that users may choose to disable the legend.
This is possible since the charts we provide already show a contextual legend when hovering a particular element of the chart, so the external legend isn't required to understand the chart.